setup.json

```diff
@@ -68,7 +68,7 @@
             "prospector==1.1.7",
             "pylint==1.9.4; python_version<'3.0'",
             "pylint==2.3.1; python_version>='3.0'",
-            "pgtest==1.2.0",
+            "pgtest~=1.3,>=1.3.1",
```

Comment:
Are you sure ``1.3.1`` fixes all the bugs, and ``1.3.2`` is not required?

Reply:
This is what we use for `aiida-core` at least

Reply:
@ltalirz should know

Reply:
Looking at the [commits](https://github.com/jamesnunn/pgtest/commits/master) after `1.3.1` they do not seem like critical fixes

Reply:
Ah, from looking at the diff 1.3.1 could potentially raise some `ResourceWarning`s, but that shouldn't break anything.

Reply:
yeah... 1.3.2 is safer but 1.3.1 will still work.